wpengine-phpcompat.php

```diff
@@ -73,21 +73,23 @@ public static function init() {
 	 * @return null
 	 */
 	function start_test() {
-		global $wpdb;
+		if ( current_user_can( 'edit_posts' ) ) {
```

Comment:
Contributors should be able to run the compatibility scan? Does the scan disclose information they don't have access to?